tomcat/urls

```diff
@@ -0,0 +1 @@
+https://archive.apache.org/dist/tomcat/tomcat-${major}/v${version}/src/apache-tomcat-${version}-src.${ext}
```

Comment:
Did you try to download tomcat manually, unpack and start it?
`src` stands for source(-code). Therefore this download URL leads to an archive containing the Java source code files of tomcat that need to be compiled and build before you can run them.
You should use `bin` instead of `src` and if I am not mistaken, you can use the regular zip distribution of tomcat that can run on all platforms so in `ext-mappings` you can just have one line `*=.zip`.